.github/workflows/release.yml

```diff
@@ -57,11 +51,45 @@ jobs:
         name: enarx-${{ matrix.platform.target }}-oci
         path: enarx-${{ matrix.platform.target }}-oci
 
+  aarch64-build:
+    strategy:
+      matrix:
+        platform:
+        - host: ARM64
+          target: aarch64-unknown-linux-musl
+        - host: aarch64-apple-darwin
+          target: aarch64-apple-darwin
+    runs-on: ${{ matrix.platform.host }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Get rust toolchain snapshot version
+        id: toolchain_version
+        run: echo "::set-output name=toolchain_version::$(grep channel  rust-toolchain.toml | cut -d\" -f 2)"
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: ${{ steps.toolchain_version.outputs.toolchain_version }}
+          target: ${{ matrix.platform.target }}
+          profile: minimal
+      - name: Setup Rust toolchain
+        run: >
+            rustup override unset || :
+            && rustup show
```

Comment:
What does this do?

Reply:
Unsets the override in cargo/config and provides a workaround in a component install bug in the github action. Toolchain and target wanted to default to another target without this defined.